cfme/tests/cloud_infra_common/test_vm_instance_analysis.py

```diff
@@ -989,3 +989,19 @@ def test_ssa_multiple_vms(ssa_multiple_vms, soft_assert, appliance, compare_linu
             compare_linux_vm_data(ssa_vm)
         else:
             compare_windows_vm_data(ssa_vm)
+
+
+@pytest.mark.manual("manualonly")
+def test_ssa_vm_ec2_agent_tracker():
+    """
+    Bugzilla:
+        1557452
+
+    Polarion:
+        assignee: sbulage
+        casecomponent: SmartState
+        initialEstimate: 1/3h
+        startsin: 5.9
+        tags: smartstate
+    """
```

Comment:
- startsin can change to **startsin: 5.10**
- Can you please add **teststeps** and **expectedResults** as this is the ```manualonly``` test case

Reply:
okay